presto-native-execution/presto_cpp/presto_protocol/presto_protocol.h

```diff
@@ -50,36 +50,39 @@ extern const char* const PRESTO_BUFFER_COMPLETE_HEADER;
 extern const char* const PRESTO_MAX_WAIT_DEFAULT;
 extern const char* const PRESTO_MAX_SIZE_DEFAULT;
 
-
 extern const char* const PRESTO_ABORT_TASK_URL_PARAM;
 
 class Exception : public std::runtime_error {
  public:
-  explicit Exception(const std::string& message) : std::runtime_error(message) {};
+  explicit Exception(const std::string& message)
+      : std::runtime_error(message){};
```

Comment:
There are so many formatting changes and it's difficult to see what's actually changed. Would it be possible to exclude the pure formatting changes?

Reply:
@yingsu00 These files are auto-generated by running `make protocol`.